lib/error.g

```diff
@@ -59,16 +59,47 @@ end);
 
 ErrorLVars := fail;
 
-BIND_GLOBAL("WHERE", function( context, depth, outercontext)
-    local   bottom,  lastcontext,  f;
+BIND_GLOBAL("PRETTY_PRINT_VARS", function(context)
+    local vars, i;
+    vars := ContentsLVars(context);
+    if IsRecord(vars) then
+        PrintTo(ERROR_OUTPUT, "-Arguments:\n");
+        for i in [1..ABS_RAT(NumberArgumentsFunction(vars.func))] do
```

Comment:
`ABS_RAT(NumberArgumentsFunction(vars.func))` is used twice, perhaps store it in a local var `numArgs` or so?